sentry_sdk/client.py

```diff
@@ -581,6 +581,17 @@ def capture_event(
 
         return event_id
 
+    def is_sentry_url(self, url):
```

Comment:
@sentrivana @szokeasaurusrex sorry to nitpick, but try to do stuff like this in utils/helpers, hub/client has a specific api which we shouldn't add to. Can you open another PR and move this to either `utils.py` or just another local function in `stdlib.py`?

Reply:
Just opened #2304 to resolve this